markdown.go

```diff
@@ -1082,6 +1089,21 @@ func expandTabs(out *bytes.Buffer, line []byte, tabSize int) {
 	}
 }
 
+// Extract indent finds the appropriate indentation for the next block
+// where line can be inside of a list.
+func extractIndent(line []byte, tabSize int) []byte {
```

Comment:
make (more) sense now. I kinda want to use this new code in other places as well.